### ISSUE TITLE
Named volume "db_sqlite3:/app/db.sqlite3:rw" no declaration was found in the volumes section.

### ISSUE THREAD

**Issue by tom729** [WongSaang/chatgpt-ui#212]
docker-compose -f docker-compose.yml up -d
ERROR: Named volume "db_sqlite3:/app/db.sqlite3:rw" is used in service "backend-wsgi-server" but no declaration was found in the volumes section.

**Comment by zam89** [comment]
yup. i also having same problem here. any help?

**Comment by zam89** [comment]
i've add suggested line as follows:
```
<snip>
networks:
  chatgpt_ui_network:
    driver: bridge

volumes:
  db_sqlite3:
```

now it can run:
```
A@A-vm:~/chatgpt-ui$ sudo docker-compose up -d
Starting chatgpt-ui_backend-wsgi-server_1 ... done
Starting chatgpt-ui_backend-web-server_1  ... done
Starting chatgpt-ui_client_1              ... done

A@A-vm:~/chatgpt-ui$ sudo docker ps
CONTAINER ID   IMAGE                                     COMMAND                  CREATED        STATUS         PORTS                                       NAMES
ba7db71ed1f1   wongsaang/chatgpt-ui-client:latest        "node server/index.m…"   11 hours ago   Up 7 seconds   0.0.0.0:80->80/tcp, :::80->80/tcp           chatgpt-ui_client_1
e34db9fb241d   wongsaang/chatgpt-ui-web-server:latest    "/docker-entrypoint.…"   11 hours ago   Up 7 seconds   0.0.0.0:9000->80/tcp, :::9000->80/tcp       chatgpt-ui_backend-web-server_1
d41d0e0073f9   wongsaang/chatgpt-ui-wsgi-server:latest   "./entrypoint.sh"        11 hours ago   Up 8 seconds   0.0.0.0:8000->8000/tcp, :::8000->8000/tcp   chatgpt-ui_backend-wsgi-server_1
```

but when I try to login as "admin" & "password", it says "Something went wrong. Please try again."

**Comment by daviddahl** [comment]
Yes, the default docker compose configuration does not work. I have tweaked it to use postgres and i am seeing 400 errors doing anything in the UI.

**Comment by tom729** [comment]
> Yes, the default docker compose configuration does not work. I have tweaked it to use postgres and i am seeing 400 errors doing anything in the UI.
> 
> I am able to login as `admin`... however, the django admin site is broken as it is missing all of its css, js, static content. I assume I need to add an API key there as there is no place in the UI to do so- was there in the past?
> 
> ```
> chatgpt-ui-backend-wsgi-server-1  | 172.20.0.3 - - [02/Jun/2023:18:56:41 +0000] "POST /api/conversation/ HTTP/1.0" 400 41 "http://0.0.0.0/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36"
> chatgpt-ui-backend-web-server-1   | 172.20.0.4 - - [02/Jun/2023:18:56:41 +0000] "POST /api/conversation/ HTTP/1.1" 400 41 "http://0.0.0.0/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36" "-"
> ```
> 
> my example docker-compose.yml: https://gist.github.com/daviddahl/01eac63a0cd594d263a0c3dabd29de3c

thanks, you help me out!

**Comment by Freshield** [comment]
just change the ./db_sqlite3:/app/db.sqlite3 to ./db.sqlite3:/app/db.sqlite3
